odl/phantom/geometric.py

```diff
@@ -95,6 +95,92 @@ def phantom(x):
     return space.element(phantom)
 
 
+def defrise(space, nellipses=8, alternating=False):
+    """Phantom with regularily spaced ellipses.
+
+    This phantom is often used to verify cone-beam algorithms.
```

Comment:
Frankly I've really looked and cannot find one, it seems to be "standard". If you google you find like half a million answers.